images/scripts/lib/bootc.bootstrap

```diff
@@ -66,3 +66,16 @@ m.execute(['podman', 'run', '--rm', '-i', '--privileged', '--security-opt=label=
 m.download('output/qcow2/disk.qcow2', args.vmpath)
 
 m.kill()
+
+# it's too small by default
+subprocess.check_call(['qemu-img', 'resize', '-f', 'qcow2', args.vmpath, '+20G'])
+
+m = testvm.VirtMachine(image=os.path.abspath(args.vmpath), maintain=True, verbose=True)
```

Comment:
Did you look into if we can do that offline with something like `systemd-repart`?  It supports operating on image files...

Reply:
No, I didn't. But `systemd-repart` only grows the partition, we need to boot the VM for `systemd-growfs`. But even then, we need to boot the VM *anyway* for copying in the bootc tar file. So while there could be "new toys", the actual optimization potential is rather low..